src/ui/layout/Footer.js

```diff
@@ -20,12 +21,19 @@ const Footer = props => {
     }
 
     return (
-        <div style={style}>
-            <span style={{fontWeight: 900}}>EAM Light</span>
-            <span style={{marginLeft: 5, marginRight: 5}}>(<span style={{textDecorationLine: "underline", cursor: "pointer"}}  onClick={() => history.push("/releaseNotes")}>v{version}</span>)</span>
-            {isCernMode && <a style={{color: "white", marginRight: 10}} href="mailto:eam.support@cern.ch">eam.support@cern.ch</a>}
-        </div>
-    )
+      <div style={style}>
+        <EAMFooter
+          appName={
+            "EAM Light " +
+            (applicationData.EL_ENVIR !== "PROD"
+              ? applicationData.EL_ENVIR
+              : "")
+          }
+          version={version}
+          supportEmail={isCernMode && "eam.support@cern.ch"}
+        />
```

Comment:
With the latest version of EAMFooter we are now missing the prop `releaseNotesPath` (and the necessary route/variable export adjustments)